lib/ecto/multi.ex

```diff
@@ -99,6 +99,7 @@ defmodule Ecto.Multi do
   defstruct operations: [], names: MapSet.new
 
   @type run :: (t -> {:ok | :error, any}) | {module, atom, [any]}
+  @type run_changeset :: (Map.t -> Changeset.t | Ecto.Schema.t)
```

Comment:
I struggled with naming this as well. I have updated to `changeset_fun` for now, but I can change again if someone has a better suggestion.